sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -26,7 +26,7 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
     super.beforeAll()
     sql(
       """
-        |CREATE EXTERNAL TABLE parquet_tab1 (c1 INT, c2 STRING)
```

Comment:
https://github.com/apache/spark/pull/12255/files#diff-1bb4f7bd5a2656f48bcd3c857167a11bR206 
will issue an exception. Thus, we have to remove `EXTERNAL`